### PR TITLE
Remove counter for single selection case

### PR DESCRIPTION
**Pull request by DummyCo**
This PR removes Apply button bracers counter in case of spec's `countable` is `false` and `maxSelectable` is `1`.

### CODE REVIEW

File: matisse/src/main/java/com/zhihu/matisse/internal/ui/BasePreviewActivity.java [hunk 1]
```diff
@@ -182,6 +182,9 @@ private void updateApplyButton() {
         if (selectedCount == 0) {
             mButtonApply.setText(R.string.button_apply_disable);
             mButtonApply.setEnabled(false);
+        } else if (selectedCount == 1 && mSpec.singleSelectionModeEnabled()) {
+            mButtonApply.setText(R.string.button_apply_disable);
```
Comment (gejiaheng) [inline matisse/src/main/java/com/zhihu/matisse/internal/ui/BasePreviewActivity.java]: I think we should rename 'R.string.button_apply_disable' to 'R.string.button_apply_default'.

Comment (DummyCo) [inline matisse/src/main/java/com/zhihu/matisse/internal/ui/BasePreviewActivity.java]: Indeed

**Comment by DummyCo** [comment]
`Add Russian and Ukrainian localization`
Also here

File: matisse/src/main/res/values-ru/strings.xml [hunk 2]
```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (gejiaheng) [inline matisse/src/main/res/values-ru/strings.xml]: It's very nice of you to add Russian and Ukrainian strings. But would you please add these localizations for the sample app?

Comment (DummyCo) [inline matisse/src/main/res/values-ru/strings.xml]: Here it is. I apologize for merging these two separate enhancements in one PR.

Comment (gejiaheng) [inline matisse/src/main/res/values-ru/strings.xml]: It's all right. You're always welcome to contribute.

**Pull request merged**